components/movementsensor/gpsnmea/pmtkI2C.go

```diff
@@ -243,8 +243,9 @@ func (g *PmtkI2CNMEAMovementSensor) Orientation(ctx context.Context, extra map[s
 // Properties what can I do!
 func (g *PmtkI2CNMEAMovementSensor) Properties(ctx context.Context, extra map[string]interface{}) (*movementsensor.Properties, error) {
 	return &movementsensor.Properties{
-		LinearVelocitySupported: true,
-		PositionSupported:       true,
+		LinearVelocitySupported:     true,
+		PositionSupported:           true,
+		LinearAccelerationSupported: true,
```

Comment:
Same thing here: the linear acceleration gives back the "unimplemented" error, so it is not supported here.